rholang/src/main/scala/coop/rchain/rholang/interpreter/storage/ChargingRSpace.scala

```diff
@@ -48,10 +46,8 @@ object ChargingRSpace {
         Option[(ContResult[Par, BindPattern, TaggedContinuation], Seq[Result[ListParWithRandom]])]
       ] =
         for {
-          _ <- charge[F](storageCostConsume(channels, patterns, continuation))
-          peekChannels = if (peek) {
-            SortedSet((0 to channels.size - 1): _*)
-          } else SortedSet.empty[Int]
+          _            <- charge[F](storageCostConsume(channels, patterns, continuation))
+          peekChannels = if (peek) SortedSet(channels.indices: _*) else SortedSet.empty[Int]
```

Comment:
Are you aware that this is a change in what the `SortedSet` looks like?
```
val c = Seq(1,2,3)
0 to c.size - 1 => Range 0 to 2
c.indices => Range 0 to 3
```

Reply:
From my repl:
```
val c = Seq(1,2,3)
0 to c.size - 1  
c.indices 
res0 == res1

//results

c: Seq[Int] = List(1, 2, 3)
res0: scala.collection.immutable.Range.Inclusive = Range 0 to 2
res1: scala.collection.immutable.Range = Range 0 until 3
res2: Boolean = true
```

So the ranges are equal, despite having different ends (one is inclusive) :)

Reply:
Thx for being vigilant! :+1: